src/queue.rs

```diff
@@ -1,35 +1,37 @@
 use libc::{c_int, size_t, c_long, ssize_t};
 use std::default::Default;
-use std::os;
+use std::io::Error;
```

Comment:
Crap, I realize I had to check these are the right ones… I guessed, I don't _know_!